{"name": "Невозможность авторизоваться курьером с некорректным значением поля", "status": "passed", "steps": [{"name": "Отправка запроса на авторизацию курьера с некорректными данными", "status": "passed", "start": 1721243814465, "stop": 1721243814542}], "parameters": [{"name": "key", "value": "'password'"}, {"name": "wrong_field", "value": "'Wrong_pass123!'"}], "start": 1721243814465, "stop": 1721243814543, "uuid": "511c5dc5-3ecb-4a52-8dc1-139d6652051a", "historyId": "f2bb0b627f0dcb3d847f3283cdd0a922", "testCaseId": "95bfda5ea90692b56bb543bf0b3a4bfb", "fullName": "tests.test_login_courier.TestLoginCourier#test_log_in_courier_with_wrong_field_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-VR12VM9"}, {"name": "thread", "value": "13348-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}